cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
```

Comment:
What convention should we follow with the generated examples files? 🤔 
Should we keep them or replace them with a readme-link?

Reply:
Yeah readme-link is probably better, gonna add one and delete these examples.

- [ ] Delete examples and provide link to them.